crypto/crypto_aes.c

```diff
@@ -89,30 +91,32 @@ useaesni(void)
 #endif /* CPUSUPPORT_X86_AESNI */
 
 /**
- * crypto_aes_key_expand(key, len):
- * Expand the ${len}-byte AES key ${key} into a structure which can be passed
- * to crypto_aes_encrypt_block.  The length must be 16 or 32.
+ * crypto_aes_key_expand(key, keylen):
```

Comment:
Well, I wanted to standardize the names so that I could make exactly the same change in both functions.  You've got

    static int
    aesnitest(uint8_t ptext[16], uint8_t * key, size_t keylen)

and

    struct crypto_aes_key *
    crypto_aes_key_expand(const uint8_t * key, size_t len)

I found the `keylen` name slightly more informative than `len`.  Granted, since the only other argument to `crypto_aes_key_expand` is `key`, I wouldn't say that `len` is a terrible name.  But changing `aesnitest` to use `len` seemed like a bad idea, since then it wouldn't be obvious at first glance whether `len` referred to `ptext` or `key`.

I'm not wedded to this change, though.